content/venues/2022/09-30/readme.md

```diff
@@ -0,0 +1,13 @@
+---
+dateStart: 2022-09-30T15:00:00.553Z
+dateEnd: 2022-09-30T16:00:00.568Z
+title: Science Maps and Macroscopes
+venue: University of Michigan Clark Library
+organizer: Timothy Utter
+credit: Places & Spaces
+city: Ann Arbor
+state: Michigan
+country: United States
+pdfLink:science-maps-and-macroscopes.pdf
```

Comment:
@ezengels invalid yaml. which causes the build to crash. pdfLink needs a space after the colon. Also the newline below is invalid. We use the Netlify CMS to avoid these errors.